### PR TITLE
Dtsstci 68

### PR DESCRIPTION
**Pull request by gmmagruder**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/DTSSTCI-68

### Change description ###
- Added aria label to delete link in upload documents for voice over
- Imported upload-documents css so it will be used

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/steps/edge-case/upload-appeal-form/template.njk [hunk 1]
```diff
@@ -32,9 +32,9 @@
     {%if uploadedDocuments.length >0 %}
     {% for file in uploadedDocuments %}
       <li class="uploadedFile govuk-!-padding-top-2 govuk-!-padding-bottom-3 govuk-section-break govuk-section-break--visible">
-        {{ file.fileName }}
+        <p class="govuk-body" id="Document{{ loop.index0 }}">{{ file.fileName }}</p>
         {% set documentId = file.documentId%}
-        <a href="/upload-appeal-form?query=delete&docId={{documentId}}&documentType=tribunalform" class="govuk-link--no-visited-state float-right ">{{ delete }}</a>
+        <a href="/upload-appeal-form?query=delete&docId={{documentId}}&documentType=tribunalform" class="govuk-link--no-visited-state float-right " id="Delete{{ loop.index0 }}" aria-labelledby="Delete{{ loop.index0 }} Document{{ loop.index0 }}">{{ delete }}</a>
```
Comment (gmmagruder) [inline src/main/steps/edge-case/upload-appeal-form/template.njk:37]: Changes for these could have been made in assets/js/upload-manager/updateFileList.ts but it was not being used. I think it used for uploading to /document-manager (DOCUMENT_UPLOAD_URL) but the code doesn't currently do that at any point.

**Review by tomxelliott — APPROVED** [review]
